plugins/browser-plugin-vimeo-tracking/src/index.ts

```diff
@@ -0,0 +1,3 @@
+export { VimeoEvent } from './events';
+export { VimeoTrackingConfiguration } from './types';
+export { startVimeoTracking, endVimeoTracking, VimeoTrackingPlugin } from './initialization';
```

Comment:
Should we reexport the `trackXX` APIs from the media plugin that are not auto-tracked by the Vimeo plugin? In particular, I think these would make sense:

* Ad events: trackMediaAdBreakEnd, trackMediaAdBreakStart, trackMediaAdClick, trackMediaAdComplete, trackMediaAdFirstQuartile, trackMediaAdMidpoint, trackMediaAdPause, trackMediaAdResume, trackMediaAdSkip, trackMediaAdStart, trackMediaAdThirdQuartile
* Custom events: trackMediaSelfDescribingEvent

Reply:
That seems sensible, prevents any confusion from importing from 2 separate sources.